ddtrace/settings/config.py

```diff
@@ -210,6 +210,8 @@ def header_tag_name(self, header_name):
         # type: (str) -> Optional[str]
         """
         Returns the tag associated with the current header if it should be traced.
+        An empty string is returned when the current header must be retained but
+        no tag was configured.
```

Comment:
Is it possible to somehow reduce these duplicate docstrings by abstracting (e.g. a "header" mixin)?

Reply:
Probably yes but this method is temporary and should be removed with https://github.com/DataDog/dd-trace-py/pull/2762.